app/brave_generated_resources.grd

```diff
@@ -172,6 +172,9 @@
       <message name="IDS_OPEN_GUEST_PROFILE" desc="The app menu item to create a new User Profile">
         Open Guest Window
       </message>
+      <message name="IDS_OPEN_MORE_OTHER_DEVICES_SESSIONS" desc="The history sub-menu item to show sessions from other devices">
+        More...
```

Comment:
Please use the ellipsis character `…`

Reply:
@petemill  this is fixed at https://github.com/brave/brave-core/pull/7121/files#diff-6314ab9f5910c0e3dd53f8ac6bf09b584020abe6790abe7ebcdacc5756c5d5f9R176